lib/Ed25519VerificationKey2020.js

```diff
@@ -293,3 +332,16 @@ function _isEqualBuffer(buf1, buf2) {
   }
   return true;
 }
+
+// check a multibase key for an expected header
+function _isValidKeyHeader(multibaseKey, expectedHeader) {
+  if((typeof multibaseKey !== 'string') ||
+    (multibaseKey[0] !== MULTIBASE_BASE58BTC_HEADER)) {
+    return false;
+  }
+
+  const keyBytes = base58btc.decode(multibaseKey.slice(1));
+  return !expectedHeader.some((val, i) => {
+    return keyBytes[i] !== val;
+  });
```

Comment:
I can't remember if there's a quicker way/cleaner way to compare byte arrays, but regardless, this is cleaner than what's there now:

```suggestion
  return expectedHeader.every((val, i) => keyBytes[i] === val);
```